packages/data/src/namespace-store/index.js

```diff
@@ -26,9 +26,11 @@ import * as metadataActions from './metadata/actions';
 /**
  * Creates a namespace object with a store derived from the reducer given.
  *
- * @param {string} key              Identifying string used for namespace and redex dev tools.
- * @param {Object} options          Contains reducer, actions, selectors, and resolvers.
- * @param {Object} registry         Registry reference.
+ * @param {string}         key      Unique namespace identifier.
+ * @param {Object}         options  Registered store options, with properties
```

Comment:
Type `Object` is not useful IMHO. I assume this is temporary until the typescript checking makes its way into this file, yes?

Reply:
> Type `Object` is not useful IMHO. I assume this is temporary until the typescript checking makes its way into this file, yes?

I'd say temporary, yes. I'm not really changing anything here aside from alignment and description. `WPDataRegistry` is the only type being changed throughout this pull request.